zos_concepts/certificate_management/README.md

```diff
@@ -1,104 +1,104 @@
+# Managed z/OS Certificates
+
+This playbook demonstrates how to copy data to and from a z/OS system using modules included in the Red Hat Ansible Certified Content for IBM Z core collection.
+
+This project contains playbooks and roles that demonstrates certificate renewal
+using RACF. The playbooks in this project are designed to address an end to end
+scenario managing z/OS certificates beginning with monitoring certificates using
+Health Checker, creating an authority and certificates, deleting certificates
+and even renewing them.
+
+ALthough this is an end to end scenario, you can choose to run the individual
+playbooks if you are interested in one particular operation. Review the individual
+playbook for more details.
+
+In addition to providing the various operations, this project goes further and
+demonstrates how some of the same operations can be run various ways. For example,
+you will notice that the two modules `zos_mvs_raw` and `zos_tso_command` are
+heavily used to perform the same operation. You can choose to create a certificate
+authority using the playbook based on `zos_mvs_raw` which is
+[**create_CERTAUTH_cert_raw.yml**](create_CERTAUTH_cert_raw.yml) or you could
+choose the playbook leveraging TSO commands to do the same thing
+[**create_CERTAUTH_cert_tsocmd.yml**](create_CERTAUTH_cert_tsocmd.yml). Either
+playbook will yield the same results and this offers you the opportunity to
+explore multiple ways to do the same operation.
+
+These playbook use:
+
+    collection:
+        ibm.ibm_zos_core
+    modules:
+        zos_mvs_raw
+        zos_tso_command
+        zos_operator
+        zos_job_submit
+
+It is a good practice to review the playbook contents before executing them.
+It will help you understand the requirements in terms of space, location, names,
+authority, and the artifacts that will be created and cleaned up.
+
+## Playbook Requirements
+This playbook requires:
+
+- [IBM® z/OS® core collection 1.3.1 or later](https://galaxy.ansible.com/ibm/ibm_zos_core)
+- [Ansible® 2.9 or 2.11](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+
+## Configuration
+- Configure the included [inventory.yml](inventories/inventory.yml) with the
+  information from the managed z/OS host.
+  - Review [inventory documentation](../docs/share/zos_core/configure_inventory.md)
+- Configure the included **host_vars** [zos_host.yml](inventories/host_vars/zos_host.yml)
+  with the information from your z/OS system.
+  - Review [host_vars documentation](../../docs/share/zos_core/configure_host_vars.md)
+    and any additional noted variables in the configuration.
+
+## Run the playbook
+This project has several playbooks that you can run, choose a `playbook-name`
+and substitute it in the command below to execute it.
 
-## Role Summary
+```bash
+ansible-playbook -i inventories <playbook-name>
```

Comment:
should be 'ansible-playbook -i ./inventory.yml <playbook-name>' to match the directory/file structure

Reply:
@adn1107 

Its a good point, we went with the Ansible Alternative directory layout and the above has been working `ansible-playbook -i inventories <playbook-name>` maybe because there is only one `inventory` in the `inventories` directory, but we could more concrete over relative and use `./inventory.yml` , let me know your thoughts so we can close this out. 

Here is the [Ansible Alternative directory layout](https://docs.ansible.com/ansible/2.8/user_guide/playbooks_best_practices.html#alternative-directory-layout)  we are following.

Reply:
I'm fine with whichever way requires less work for the users

Reply:
ok, we might want to document that more concrete way, i am interested in knowing what would happen if we have more than one inventory file, guess that is a test I can do and then update all the playbooks on that finding not just this playbook.